src/styles/examples.scss

```diff
@@ -158,6 +158,9 @@ article {
 
   .chart {
     margin-right: 200px;
+    & .rv-xy-plot__axis__tick__line {
+      stroke: #6b6b76;
```

Comment:
nits: 
- i dont think you need the &
- can you use a scss variable instead for the hex value?